[2021-03-25 19:54:33] local.ERROR: symlink(): Protocol error {"exception":"[object] (ErrorException(code: 0): symlink(): Protocol error at /home/vagrant/code/tdd/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php:315)
[stacktrace]
#0 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleError()
#1 /home/vagrant/code/tdd/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(315): symlink()
#2 /home/vagrant/code/tdd/vendor/laravel/framework/src/Illuminate/Foundation/Console/StorageLinkCommand.php(45): Illuminate\\Filesystem\\Filesystem->link()
#3 /home/vagrant/code/tdd/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): Illuminate\\Foundation\\Console\\StorageLinkCommand->handle()
#4 /home/vagrant/code/tdd/vendor/laravel/framework/src/Illuminate/Container/Util.php(40): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#5 /home/vagrant/code/tdd/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(93): Illuminate\\Container\\Util::unwrapIfClosure()
#6 /home/vagrant/code/tdd/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(37): Illuminate\\Container\\BoundMethod::callBoundMethod()
#7 /home/vagrant/code/tdd/vendor/laravel/framework/src/Illuminate/Container/Container.php(611): Illuminate\\Container\\BoundMethod::call()
#8 /home/vagrant/code/tdd/vendor/laravel/framework/src/Illuminate/Console/Command.php(136): Illuminate\\Container\\Container->call()
#9 /home/vagrant/code/tdd/vendor/symfony/console/Command/Command.php(256): Illuminate\\Console\\Command->execute()
#10 /home/vagrant/code/tdd/vendor/laravel/framework/src/Illuminate/Console/Command.php(121): Symfony\\Component\\Console\\Command\\Command->run()
#11 /home/vagrant/code/tdd/vendor/symfony/console/Application.php(971): Illuminate\\Console\\Command->run()
#12 /home/vagrant/code/tdd/vendor/symfony/console/Application.php(290): Symfony\\Component\\Console\\Application->doRunCommand()
#13 /home/vagrant/code/tdd/vendor/symfony/console/Application.php(166): Symfony\\Component\\Console\\Application->doRun()
#14 /home/vagrant/code/tdd/vendor/laravel/framework/src/Illuminate/Console/Application.php(92): Symfony\\Component\\Console\\Application->run()
#15 /home/vagrant/code/tdd/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(129): Illuminate\\Console\\Application->run()
#16 /home/vagrant/code/tdd/artisan(37): Illuminate\\Foundation\\Console\\Kernel->handle()
#17 {main}
"} 
[2021-03-25 20:58:40] laravel.EMERGENCY: Unable to create configured logger. Using emergency logger. {"exception":"[object] (InvalidArgumentException(code: 0): Log [] is not defined. at D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\vendor\\laravel\\framework\\src\\Illuminate\\Log\\LogManager.php:192)
[stacktrace]
#0 D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\vendor\\laravel\\framework\\src\\Illuminate\\Log\\LogManager.php(118): Illuminate\\Log\\LogManager->resolve(NULL)
#1 D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\vendor\\laravel\\framework\\src\\Illuminate\\Log\\LogManager.php(98): Illuminate\\Log\\LogManager->get(NULL)
#2 D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\vendor\\laravel\\framework\\src\\Illuminate\\Log\\LogManager.php(555): Illuminate\\Log\\LogManager->driver()
#3 D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Exceptions\\Handler.php(250): Illuminate\\Log\\LogManager->error('syntax error, u...', Array)
#4 D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Console\\Kernel.php(366): Illuminate\\Foundation\\Exceptions\\Handler->report(Object(ParseError))
#5 D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Console\\Kernel.php(131): Illuminate\\Foundation\\Console\\Kernel->reportException(Object(ParseError))
#6 Command line code(1): Illuminate\\Foundation\\Console\\Kernel->handle(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))
#7 {main}
"} 
[2021-03-25 20:58:40] laravel.ERROR: syntax error, unexpected 'Features' (T_STRING), expecting ']' {"exception":"[object] (ParseError(code: 0): syntax error, unexpected 'Features' (T_STRING), expecting ']' at D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\config\\jetstream.php:49)
[stacktrace]
#0 D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Bootstrap\\LoadConfiguration.php(39): Illuminate\\Foundation\\Bootstrap\\LoadConfiguration->loadConfigurationFiles(Object(Illuminate\\Foundation\\Application), Object(Illuminate\\Config\\Repository))
#1 D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Application.php(237): Illuminate\\Foundation\\Bootstrap\\LoadConfiguration->bootstrap(Object(Illuminate\\Foundation\\Application))
#2 D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Console\\Kernel.php(310): Illuminate\\Foundation\\Application->bootstrapWith(Array)
#3 D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Console\\Kernel.php(127): Illuminate\\Foundation\\Console\\Kernel->bootstrap()
#4 Command line code(1): Illuminate\\Foundation\\Console\\Kernel->handle(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))
#5 {main}
"} 
[2021-03-25 20:30:20] local.ERROR: syntax error, unexpected '}', expecting '(' {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): syntax error, unexpected '}', expecting '(' at D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\app\\Models\\User.php:64)
[stacktrace]
#0 {main}
"} 
[2021-03-25 20:30:40] local.ERROR: syntax error, unexpected '}', expecting '(' {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): syntax error, unexpected '}', expecting '(' at D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\app\\Models\\User.php:64)
[stacktrace]
#0 {main}
"} 
[2021-03-25 20:31:40] local.ERROR: syntax error, unexpected '}', expecting '(' {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): syntax error, unexpected '}', expecting '(' at D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\app\\Models\\User.php:64)
[stacktrace]
#0 {main}
"} 
[2021-03-25 20:32:40] local.ERROR: syntax error, unexpected '}', expecting '(' {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): syntax error, unexpected '}', expecting '(' at D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\app\\Models\\User.php:64)
[stacktrace]
#0 {main}
"} 
[2021-03-25 20:33:40] local.ERROR: syntax error, unexpected '}', expecting '(' {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): syntax error, unexpected '}', expecting '(' at D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\app\\Models\\User.php:64)
[stacktrace]
#0 {main}
"} 
[2021-03-25 20:34:40] local.ERROR: syntax error, unexpected '}', expecting '(' {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): syntax error, unexpected '}', expecting '(' at D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\app\\Models\\User.php:64)
[stacktrace]
#0 {main}
"} 
[2021-03-25 20:35:23] local.ERROR: syntax error, unexpected '}', expecting ';' {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): syntax error, unexpected '}', expecting ';' at D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\app\\Models\\User.php:65)
[stacktrace]
#0 {main}
"} 
[2021-03-25 20:35:40] local.ERROR: syntax error, unexpected '}', expecting identifier (T_STRING) or variable (T_VARIABLE) or '{' or '$' {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): syntax error, unexpected '}', expecting identifier (T_STRING) or variable (T_VARIABLE) or '{' or '$' at D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\app\\Models\\User.php:65)
[stacktrace]
#0 {main}
"} 
[2021-03-26 15:00:51] local.ERROR: syntax error, unexpected '}', expecting ';' {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): syntax error, unexpected '}', expecting ';' at D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\app\\Models\\Repository.php:14)
[stacktrace]
#0 {main}
"} 
[2021-03-26 15:43:20] local.ERROR: Uncaught Error: Class 'App\Models\Model' not found in D:\juc\desaPHP\entornovirtual\code\tdd\app\Models\Repository.php:9
Stack trace:
#0 Command line code(1): include_once()
#1 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'App\\Models\\Model' not found in D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\app\\Models\\Repository.php:9
Stack trace:
#0 Command line code(1): include_once()
#1 {main}
  thrown at D:\\juc\\desaPHP\\entornovirtual\\code\\tdd\\app\\Models\\Repository.php:9)
[stacktrace]
#0 {main}
"} 
